pkg/lib/get-timesync-backend.py

```diff
@@ -33,7 +33,7 @@ def first_unit(files):
 
 if unit == "systemd-timesyncd.service":
     print("timesyncd")
-elif unit in [ "chronyd.service", "chrony.service" ] and os.path.isdir("/etc/chrony/sources.d"):
```

Comment:
Please fix the s/is/if/ typo in the commit message

Reply:
Done.